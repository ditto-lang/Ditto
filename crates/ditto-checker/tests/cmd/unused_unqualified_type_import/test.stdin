module Test exports (..);

import Five (Five);

five = Five.five;
