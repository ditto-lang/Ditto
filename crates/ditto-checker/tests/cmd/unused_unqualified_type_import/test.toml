bin.name = "ditto-checker-testbin-check-module"
args = ["unused_unqualified_type_import.ditto"]
fs.sandbox = true
