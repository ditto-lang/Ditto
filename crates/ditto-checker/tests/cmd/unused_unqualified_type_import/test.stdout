{
  "module_name": [
    "Test"
  ],
  "exports": {
    "types": {},
    "constructors": {},
    "values": {
      "five": {
        "doc_comments": [],
        "doc_position": 0,
        "value_type": {
          "type": "PrimConstructor",
          "data": "Int"
        }
      }
    }
  },
  "types": {},
  "constructors": {},
  "values": {
    "five": {
      "doc_comments": [],
      "name_span": {
        "start_offset": 46,
        "end_offset": 50
      },
      "expression": {
        "expression": "ImportedVariable",
        "data": {
          "span": {
            "start_offset": 53,
            "end_offset": 62
          },
          "variable_type": {
            "type": "PrimConstructor",
            "data": "Int"
          },
          "variable": {
            "module_name": [
              null,
              [
                "Five"
              ]
            ],
            "value": "five"
          }
        }
      }
    }
  },
  "values_toposort": [
    "five"
  ]
}